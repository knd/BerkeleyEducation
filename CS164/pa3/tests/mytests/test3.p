reverse([A|B],Z,W) :- reverse(B,[A|Z],W).
reverse([],A,A).
reverse([1,2,3,4,5],[],X)?
