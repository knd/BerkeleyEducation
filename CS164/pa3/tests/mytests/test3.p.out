Asking for solution 1
X = [5|[4|[3|[2|[1|[]]]]]]
Asking for solution 2
None
